README.md

```diff
@@ -251,23 +277,29 @@ Read in a different language: [![CN](./assets/flags/CN.png)**CN**](./README.chin
 
-**Otherwise:** Your API will be accessible for testing via HTTP calls only (slower and much harder to generate coverage reports). It probably won't be a big pleasure to maintain hundreds of lines of code in a single file
+**Otherwise:** Failing to satisfy any of the config requirements will simply bog down the development or DevOps team. Probably both
 
-🔗 [**Read More: separate Express 'app' and 'server'**](./sections/projectstructre/separateexpress.md)
+🔗 [**Read More: configuration best practices**](./sections/projectstructre/configguide.md)
 
-<br/><br/>
+## ![✔] 1.5 Consider all the consequences when choosing the main framework
 
-## ![✔] 1.5 Use environment aware, secure and hierarchical config
+**TL;DR:** When building apps and APIs, using a framework is mandatory. It's easy to overlook alternative frameworks or important considerations and then finally land on a sub optimal option. As of 2023/2024, we believe that these four frameworks are worth considering: [Nest.js](https://nestjs.com/), [Fastify](https://www.fastify.io/), [express](https://expressjs.com/), and [Koa](https://koajs.com/). Click read more below for a detailed pros/cons of each framework. Simplistically, we believe that Nest.js is the best match for teams who wish to go OOP and/or build large-scale apps that can't get partitioned into smaller _autonomous_ components. Fastify is our recommendation for apps with reasonably-sized components (e.g., Microservices) that are built around simple Node.js mechanics. Read our [full considerations guide here](./sections/projectstructre/choose-framework.md)
 
-**TL;DR:** A perfect and flawless configuration setup should ensure (a) keys can be read from file AND from environment variable (b) secrets are kept outside committed code (c) config is hierarchical for easier findability. There are a few packages that can help tick most of those boxes like [rc](https://www.npmjs.com/package/rc), [nconf](https://www.npmjs.com/package/nconf), [config](https://www.npmjs.com/package/config), and [convict](https://www.npmjs.com/package/convict).
+**Otherwise:** Due to the overwhelming amount of considerations, it's easy to make decisions based on partial information and compare apples with oranges. For example, it's believed that Fastify is a minimal web-server that should get compared with express only. In reality, it's a rich framework with many official plugins that cover many concerns
 
-**Otherwise:** Failing to satisfy any of the config requirements will simply bog down the development or DevOps team. Probably both
+🔗 [**Read More: configuration best practices**](./sections/projectstructre/choose-framework.md)
 
-🔗 [**Read More: configuration best practices**](./sections/projectstructre/configguide.md)
+## ![✔] 1.6 Use TypeScript sparingly and thoughtfully
```

Comment:
I think you should not recommend specific language here as you actually talk about type safety so maybe change the title to match the type safety point?